Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs

```diff
@@ -100,6 +100,7 @@ public struct UnityMaterial {
   /// <param name="loader">The loader to use to load resources (textures, etc).</param>
   /// <param name="loaded">Mutated to add any textures that were loaded.</param>
   public static IEnumerable LoadTexturesCoroutine(
+      if (gltfTexture.SourcePtr.uri == null) continue;
```

Comment:
It doesn't look like our unity build likes the syntax here. Did something get lost between the development and the commit?

```
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(102,51): error CS1041: Identifier expected; 'if' is a keyword
[1750](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1750)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,37): error CS8124: Tuple must contain at least two elements.
[1751](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1751)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,37): error CS1026: ) expected
[1752](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1752)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,37): error CS1001: Identifier expected
[1753](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1753)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,37): error CS1003: Syntax error, ',' expected
[1754](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1754)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,46): error CS1002: ; expected
[1755](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1755)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(103,46): error CS1519: Invalid token 'continue' in class, struct, or interface member declaration
[1756](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1756)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(104,26): error CS1044: Cannot use more than one type in a for, using, fixed, or declaration statement
[1757](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1757)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(104,37): error CS1002: ; expected
[1758](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1758)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(104,43): error CS1519: Invalid token ',' in class, struct, or interface member declaration
[1759](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1759)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(104,67): error CS1003: Syntax error, ',' expected
[1760](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1760)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,13): error CS1002: ; expected
[1761](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1761)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,42): error CS8124: Tuple must contain at least two elements.
[1762](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1762)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,42): error CS1026: ) expected
[1763](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1763)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,42): error CS1519: Invalid token 'in' in class, struct, or interface member declaration
[1764](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1764)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,67): error CS1001: Identifier expected
[1765](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1765)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,68): error CS1002: ; expected
[1766](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1766)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(105,68): error CS1519: Invalid token ')' in class, struct, or interface member declaration
[1767](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1767)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(106,31): error CS8124: Tuple must contain at least two elements.
[1768](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1768)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(106,31): error CS1026: ) expected
[1769](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1769)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(106,57): error CS8124: Tuple must contain at least two elements.
[1770](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1770)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(106,58): error CS1519: Invalid token ')' in class, struct, or interface member declaration
[1771](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1771)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(107,31): error CS1519: Invalid token '(' in class, struct, or interface member declaration
[1772](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1772)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(107,32): error CS1031: Type expected
[1773](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1773)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(107,32): error CS8124: Tuple must contain at least two elements.
[1774](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1774)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(107,32): error CS1026: ) expected
[1775](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1775)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(107,32): error CS1519: Invalid token '"Texture {0} uri {1} was considered necessary"' in class, struct, or interface member declaration
[1776](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1776)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(108,50): error CS1519: Invalid token ',' in class, struct, or interface member declaration
[1777](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1777)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(108,77): error CS1519: Invalid token ')' in class, struct, or interface member declaration
[1778](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1778)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,7): error CS1022: Type or namespace definition, or end-of-file expected
[1779](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1779)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,27): error CS8124: Tuple must contain at least two elements.
[1780](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1780)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,27): error CS1026: ) expected
[1781](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1781)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,27): error CS1022: Type or namespace definition, or end-of-file expected
[1782](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1782)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,65): error CS1001: Identifier expected
[1783](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1783)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,73): error CS1001: Identifier expected
[1784](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1784)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,74): error CS1002: ; expected
[1785](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1785)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(111,74): error CS1022: Type or namespace definition, or end-of-file expected
[1786](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1786)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(112,9): error CS0116: A namespace cannot directly contain members such as fields or methods
[1787](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1787)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(112,15): error CS1022: Type or namespace definition, or end-of-file expected
[1788](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1788)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(114,7): error CS1022: Type or namespace definition, or end-of-file expected
[1789](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1789)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(114,36): error CS8124: Tuple must contain at least two elements.
[1790](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1790)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(114,36): error CS1026: ) expected
[1791](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1791)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(114,36): error CS1022: Type or namespace definition, or end-of-file expected
[1792](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1792)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(115,16): error CS0116: A namespace cannot directly contain members such as fields or methods
[1793](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1793)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(115,44): error CS8124: Tuple must contain at least two elements.
[1794](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1794)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(115,45): error CS1022: Type or namespace definition, or end-of-file expected
[1795](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1795)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(116,7): error CS1022: Type or namespace definition, or end-of-file expected
[1796](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1796)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(117,5): error CS1022: Type or namespace definition, or end-of-file expected
[1797](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1797)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,5): error CS1022: Type or namespace definition, or end-of-file expected
[1798](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1798)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,34): error CS8124: Tuple must contain at least two elements.
[1799](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1799)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,34): error CS1026: ) expected
[1800](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1800)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,34): error CS1022: Type or namespace definition, or end-of-file expected
[1801](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1801)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,42): error CS0116: A namespace cannot directly contain members such as fields or methods
[1802](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1802)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(121,48): error CS1022: Type or namespace definition, or end-of-file expected
[1803](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1803)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(122,13): error CS0116: A namespace cannot directly contain members such as fields or methods
[1804](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1804)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(122,18): error CS1022: Type or namespace definition, or end-of-file expected
[1805](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1805)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(123,5): error CS1022: Type or namespace definition, or end-of-file expected
[1806](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1806)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(124,3): error CS1022: Type or namespace definition, or end-of-file expected
[1807](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1807)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(532,1): error CS1022: Type or namespace definition, or end-of-file expected
[1808](https://github.com/icosa-gallery/open-brush/runs/6092513323?check_suite_focus=true#step:16:1808)
Assets/ThirdParty/TiltBrushToolkit/Scripts/Gltf/GltfMaterialConverter.cs(534,1): error CS1022: Type or namespace definition, or end-of-file expected
```